ukis_pysat/file.py

```diff
@@ -118,6 +118,19 @@ def get_ts_from_sentinel_filename(filename, start_date=True):
             return datetime.strptime(filename[32:47], "%Y%m%dT%H%M%S").replace(tzinfo=timezone.utc)
 
 
+def get_ESA_date_from_datetime(dt: datetime, dformat: str = "%Y%m%dT%H%M%S") -> str:
```

Comment:
I would rename the function since any datetime object, without any relation to ESA files, can be renamed with a given format.

Reply:
yes, but with the default dformat it is the ESA one. If we leave it general this step is very unnecessary.

Reply:
If there is more ESA (or NASA) formats we could add them to the docstring.

Reply:
Like [this ](https://gisgeography.com/landsat-file-naming-convention/) one.

Landsat files are named with the followinig scheme: LXS PPPRRR  YYYYDDD  GSIVV

> For example a day like February 15 would have a Julian calendar value of 046. There are 31 days in January and add the total in February. (31+15=46)
> 
> YYYY = Year
> Year of acquisition
> 
> DDD = Julian day of year
> The count of days from the start of the year

Reply:
This only makes sense if we offer the other direction as well.

Reply:
True, let's keep this in mind when introducing the functionality.